doc/tutorials/03-writing-your-first-application.asciidoc

```diff
@@ -11,21 +11,26 @@ layout: docs
 === Building the QSkinny repository
 
 In this chapter we will write a simple QSkinny application on Linux from scratch.
-As a prerequisite, a recent Qt version (>= 5.6) should be available and the directory of
-its `qmake` binary in the current `$PATH`. On debian bullseye we need to install
-these packages `build-essential qt-qmake qtbase5-dev qtbase5-private-dev
-qtdeclarative5-dev qtdeclarative5-private-dev libqt5svg5-dev`.
+As a prerequisite, a recent Qt version (>= 5.15) should be available. On debian bullseye we need to install
+these packages `build-essential cmake qtbase5-dev qtbase5-private-dev qtdeclarative5-dev qtdeclarative5-private-dev libqt5svg5-dev`.
+For Qt6 you need the corresponding packages.
 
 Then we can build and install QSkinny to `/opt/qskinny` with the following commands:
 
-[source,xml]
+[source,shell]
 ....
-cd /home/user/dev/
-git clone https://github.com/uwerat/qskinny.git
-cd qskinny
-PREFIX=/opt/qskinny qmake -r
-make
-sudo make install
+$ git clone https://github.com/uwerat/qskinny.git # clone
+$ cd qskinny
+$ mkdir build && cd build
+$ cmake ../ && make # build
+$ sudo make install # install
+....
+
+To target a specific Qt version simply pass the cmake build variable `QSK_QT_VERSION` during the build step:
+
+[source,shell]
+....
+$ cmake -DQSK_QT_VERSION=Qt5 ../ && make
```

Comment:
QSK_QT_VERSION not part of this PR and I don't understand why we need and additional variable alongside `QT_DIR`, `Qt5_DIR`, `CMAKE_PREFIX_PATH`

Reply:
This was a response to working around a bug in #355 which did not load Qt6. I am open to removing this build var.